sdk/python/lib/pulumi/runtime/proto/provider_pb2_grpc.py

```diff
@@ -103,44 +103,44 @@ def GetSchema(self, request, context):
     """GetSchema fetches the schema for this resource provider.
     """
     context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-    context.set_details('Method not implemented!')
-    raise NotImplementedError('Method not implemented!')
+    context.set_details('Method not implemented: GetSchema!')
```

Comment:
Sure, will revert. This mostly helped me in the initial phase trying to trace the gRPC. I now have some better ways to trace gRPC anyway so this can be reverted.